src/components/chat/Chat.vue

```diff
@@ -56,19 +56,24 @@
                 class=""
                 v-model="currentMessage"
               />
-              <div class="flex items-center justify-between py-2">
-                <base-link href="" text-variant="bodysm" class="px-2"
-                  >{{ $t('chat.need_help') }}
-                </base-link>
+              <div class="flex justify-between py-2">
+                <!--                <base-link href="" text-variant="bodysm" class="px-2"-->
+                <!--                  >{{ $t('chat.need_help') }}-->
+                <!--                </base-link>-->
+                <div class="mx-1 w-8 h-8">
+                  <base-checkbox v-model="urgent">
+                    {{ $t('chat.urgent') }}
+                  </base-checkbox>
+                </div>
                 <div class="flex">
-                  <ccu-icon
-                    :alt="$t('chat.urgent')"
-                    size="small"
-                    type="attention"
-                    class="mx-1 w-8 h-8"
-                    :class="urgent ? 'bg-crisiscleanup-light-smoke' : ''"
-                    @click.native="urgent = !urgent"
-                  />
+                  <!--                  <ccu-icon-->
+                  <!--                    :alt="$t('chat.urgent')"-->
+                  <!--                    size="small"-->
+                  <!--                    type="attention"-->
+                  <!--                    class="mx-1 w-8 h-8"-->
+                  <!--                    :class="urgent ? 'bg-crisiscleanup-light-smoke' : ''"-->
+                  <!--                    @click.native="urgent = !urgent"-->
+                  <!--                  />-->
```

Comment:
Why did you comment these lines and not delete them? It's better not to leave commented out code (except in some special cases) as version control already tracks your changes.

Reply:
done